integration/src/main/resources/suites/PRIntegration.json

```diff
@@ -8,6 +8,7 @@
     "integration/DeleteWorkspaceWithControlledResource.json",
     "integration/EnumerateResources.json",
     "integration/GetRoles.json",
+    "integration/ImportDataCollection.json",
```

Comment:
How long does this test take to run?  If it is long-running, it should probably be in the Full suite

Reply:
It does take a long time ~20 min. Moved to the Full suite.